core/src/main/scala/kafka/server/DelayedCreateTopics.scala

```diff
@@ -42,16 +40,16 @@ class DelayedCreateTopics(delayMs: Long,
   extends DelayedOperation(delayMs) {
```

Comment:
DelayedCreateTopics is now used for both topic and partition creation. But it's actually waiting for partitions, so maybe the name is a bit misleading now, and `DelayedCreatePartititions` is a better name.

Reply:
I think that's reasonable. I was thinking of that too, but it was pretty late so I didn't do it in the end. Will update the PR.